Sources/ITwinMobile/ITMAuthorizationClient.swift

```diff
@@ -207,6 +208,20 @@ open class ITMAuthorizationClient: NSObject, AuthorizationClient, OIDAuthStateCh
         saveState()
     }
     
+    /// Calls the onUserStateChanged callback, if that callback is set.
+    open func raiseOnUserStateChanged() {
```

Comment:
This should be `raiseOnAccessTokenChanged`.

Reply:
Fixed. I also renamed `sign(in: completion:...)` to `signIn(completion:...)`.